src/tracer.js

```diff
@@ -177,7 +176,7 @@ function sendCurrentTrace(traceSender) {
  *  */
 function postTrace(traceObject) {
     utils.debugLog(`Posting trace to ${config.getConfig().traceCollectorURL}`);
-    utils.debugLog(`trace: ${util.inspect(traceObject)}`);
+    utils.debugLog(`trace: ${JSON.stringify(traceObject, null, 2)}`);
```

Comment:
Avoid printing the whole JSON to log?

Reply:
It's annoying - if we're in debug mode I prefer to see the full payload